### PR TITLE
Fix click event not sent in addPaymentButton

### LINKED ISSUE #29310: Amount value is not displayed on payment block
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

After the merge of this [PR](https://github.com/PrestaShop/PrestaShop/pull/29180) the Amount value is not displayed on View order page > payment table
Before the merge =>  https://watch.screencastify.com/v/rNFjvPK8ZnoPHFyvsdax
After the merge => https://watch.screencastify.com/v/2q8D0nG2quV1UA5YsQWo
Capture d’écran de 2022-08-11 15-33-58

### Expected behavior

When we click on 'Enter payment button' the amount value should be displayed on the first line of payment table
Capture d’écran de 2022-08-11 15-34-12

### Steps to reproduce

1- Go to  orders > view order page
2- Go to documents tab
3- Click on create invoice
4- Click on enter payment
5- See the error
=> Amount value is not displayed on payment block also the ID of the input is deleted

### PrestaShop version(s) where the bug happened

8.0.x

### PHP version(s) where the bug happened

7.3

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by FabienPapet**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x 
| Description?      | Fix missing id
| Type?             | bug fix 
| Category?         | BO 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #29310.

### CODE REVIEW

**Review by matks — DISMISSED** [review]
@FabienPapet you can find a better PR title 😄

**Review by kpodemski — CHANGES_REQUESTED** [review]
This is probably not the best way to solve this issue.

This widget is global. I suggest adding a new option in the form type to allow developers to add their IDs.

**Comment by FabienPapet** [comment]
Updated PR & fix PR title @matks @kpodemski

**Review by kpodemski — APPROVED** [review]

File: admin-dev/themes/new-theme/js/pages/order/OrderViewPageMap.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@
 export default {
   mainDiv: '#order-view-page',
   orderPaymentDetailsBtn: '.js-payment-details-btn',
-  orderPaymentFormAmountInput: '#order_payment_amount',
+  orderPaymentFormAmountInput: '#order_payment_amount_currency_amount',
```
Comment (Amoifr) [inline admin-dev/themes/new-theme/js/pages/order/OrderViewPageMap.ts:31]: order_payment_amount_currency_amount => don't understand what it means ... without the last amount, I understand it's an order amount with currency .... but the last amount makes some blur in my mind

File: tests/UI/pages/BO/orders/view/paymentBlock.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ class PaymentBlock extends ViewOrderBasePage.constructor {
     this.paymentDateInput = '#order_payment_date';
     this.paymentMethodInput = '#order_payment_payment_method';
     this.transactionIDInput = '#order_payment_transaction_id';
-    this.paymentAmountInput = '#order_payment_amount';
+    this.paymentAmountInput = '#order_payment_amount_currency_amount';
```
Comment (Amoifr) [inline tests/UI/pages/BO/orders/view/paymentBlock.js:23]: as I said previously.

**Review by Amoifr — COMMENTED** [review]
may be it's because I don't handle the subject ... but the id may be change to be more understandable.

File: admin-dev/themes/new-theme/js/pages/order/OrderViewPageMap.ts (hunk 1 above)
Comment (FabienPapet) [reply admin-dev/themes/new-theme/js/pages/order/OrderViewPageMap.ts:31]: The id is autogenerated from the form type `amount`. 

cf OrderPaymentType AmountCurrency

Comment (Amoifr) [reply admin-dev/themes/new-theme/js/pages/order/OrderViewPageMap.ts:31]: it's sad :-(

**Review by NeOMakinG — APPROVED** [review]
🍬

**Review by matks — APPROVED** [review]

**Review by sLorenzini — APPROVED** [review]
Hello @FabienPapet,

Thank you for this PR.
Tested on 8.0.X with PHP 7.3 version.

The ID of the field is well updated and is correctly aligned:

Capture d’écran 2022-09-01 à 14 30 56

 QA approved ✅ 

Thank you

**Comment by kpodemski** [comment]
Thank you @FabienPapet @sLorenzini

**Pull request merged**